setup.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this file should be removed.

Reply:
I meant to add a comment about this. I think this is needed in order for anatomia, which is still using requirements.txt/pip installation, to install werkit from git. So, once we are ready to cut a new werkit release, I will delete this file. 

Do you think we should attempt to integrate this branch with armscyence to validate it before merging the branch/PR, and cutting a new werkit release?